build/run_python_integration_tests.py

```diff
@@ -3,6 +3,8 @@
 '''
 Runs Python integration tests
 
+Assumes  `make python src_libs && source build/py_env/bin/activate`
```

Comment:
does order matter? as in https://github.com/dnanexus/cells/pull/964/files you use `make src_libs python`

Reply:
Order does not matter.